src/Microsoft.DotNet.Configurer/DotnetFirstTimeUseConfigurer.cs

```diff
@@ -12,37 +12,59 @@ public class DotnetFirstTimeUseConfigurer
         private IEnvironmentProvider _environmentProvider;
         private INuGetCachePrimer _nugetCachePrimer;
         private INuGetCacheSentinel _nugetCacheSentinel;
+        private IFirstTimeUseNoticeSentinel _firstTimeUseNoticeSentinel;
 
         public DotnetFirstTimeUseConfigurer(
             INuGetCachePrimer nugetCachePrimer,
             INuGetCacheSentinel nugetCacheSentinel,
+            IFirstTimeUseNoticeSentinel firstTimeUseNoticeSentinel,
             IEnvironmentProvider environmentProvider)
         {
             _nugetCachePrimer = nugetCachePrimer;
             _nugetCacheSentinel = nugetCacheSentinel;
+            _firstTimeUseNoticeSentinel = firstTimeUseNoticeSentinel;
             _environmentProvider = environmentProvider;
         }
 
         public void Configure()
         {
-            if(ShouldPrimeNugetCache())
+            if (ShouldPrintFirstTimeUseNotice())
             {
                 PrintFirstTimeUseNotice();
+            }
+
+            if (ShouldPrimeNugetCache())
+            {
+                string cachePrimeMessage = LocalizableStrings.NugetCachePrimeMessage;
+                Reporter.Output.WriteLine();
+                Reporter.Output.WriteLine(cachePrimeMessage);
 
                 _nugetCachePrimer.PrimeCache();
             }
         }
 
+        private bool ShouldPrintFirstTimeUseNotice()
+        {
+            return ShouldRunFirstRunExperience() && !_firstTimeUseNoticeSentinel.Exists();
```

Comment:
I think it will be helpful if we add another environment variable (`DOTNET_DO_NOT_PRINT_TELEMETRY_MESSAGE`) or something, which could be set to allow priming the cache but not printing the message.  That way, at RPM install time, if we want to expand the cache to a system wide location, we don't accidentally show the message as well and then record that we did that.